contracts/test/benfactorJoin.js

```diff
@@ -57,8 +57,19 @@ contract('PoolDeposits', accounts => {
     await dai.approve(poolDeposits.address, mintAmount, {
       from: accounts[2],
     });
-    await poolDeposits.createProposal('Some IPFS hash string', {
-      from: accounts[2],
+    const { logs } = await poolDeposits.createProposal(
```

Comment:
This is failing. The proposalHashString we are getting is 'some IPFS hash string', but what was expected is some 0x134771824866fq989af9af0 blah blah

Reply:
I was thinking about this when I setup the graph last week. Calling this a hash string isn't probably correct, we should call it the `proposalDataIdentifier` or something. Because now we are using 3box threads, and even that we can change in the future.

Reply:
But I don't fully understand why it expected a hash when you give it a string?

Reply:
This is fixed now. Issue was I was indexing it in the event, which was giving issues. Agree on name change